pyaedt/application/Design.py

```diff
@@ -568,26 +374,35 @@ def __init__(
             self.release_on_exit = False
 
         self._mttime = None
-        assert design_type in design_solutions, "Invalid design type is specified: {}.".format(design_type)
         self._design_type = design_type
         self._desktop = main_module.oDesktop
         self._desktop_install_dir = main_module.sDesktopinstallDirectory
         self._messenger = self._logger._messenger
         self._aedt_version = self._desktop.GetVersion()[0:6]
-
-        if solution_type:
-            assert (
-                solution_type in design_solutions[design_type]
-            ), "Invalid solution type {0} exists for design type {1}.".format(solution_type, design_type)
-        self._solution_type = solution_type
-        self._oproject = None
         self._odesign = None
+        self._oproject = None
         self._design_type = design_type
+
+        if design_type == "HFSS":
+            self.design_solutions = HFSSDesignSolution(None, design_type, self._aedt_version)
+        elif design_type == "Icepak":
+            self.design_solutions = IcepakDesignSolution(None, design_type, self._aedt_version)
+        elif design_type == "Maxwell 2D":
+            self.design_solutions = Maxwell2DDesignSolution(None, design_type, self._aedt_version)
+        elif design_type == "RMxprtSolution" or design_type == "ModelCreation":
+            self.design_solutions = RmXprtDesignSolution(None, design_type, self._aedt_version)
+        else:
+            self.design_solutions = DesignSolution(None, design_type, self._aedt_version)
+        self.design_solutions._solution_type = solution_type
         self.oproject = project_name
         self.odesign = design_name
+        self.design_solutions._odesign = self.odesign
+        if solution_type:
+            self.design_solutions.solution_type = solution_type
         self._oimport_export = self._desktop.GetTool("ImportExport")
+        self._odefinition_manager = self._oproject.GetDefinitionManager()
+        self._omaterial_manager = self.odefinition_manager.GetManager("Material")
```

Comment:
@maxcapodi78 Those lines are modifying the main. (It might be due to some merge issue). 
We now have properties to handle those attributes.
```suggestion
```